### PR TITLE
add recurring autoname robot test

### PR DESCRIPTION
**Pull request by Julian88Tex**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by Julian88Tex** [comment]
**lurch: attach W-036979

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001CCR7QAO" target="_blank">W-036979</a>

File: robot/Cumulus/tests/browser/recurring_donations/check_recurring_auto_naming.robot [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    Set Suite Variable           ${ns}
+
+*** Test Cases ***
+
+Check Recurring Donation Autoname
+    [Documentation]              This test verifies that the Autonaming setting for Recurring Donations is working properly.
+
+    #Enable Autonaming Setting
+    @{recurringsettings} =       Salesforce Query                           npe03__Recurring_Donations_Settings__c
+    ...                          select=Id,EnableAutomaticNaming__c
```
Comment (skristem) [inline robot/Cumulus/tests/browser/recurring_donations/check_recurring_auto_naming.robot]: you forgot namespace for the EnableAutomaticNaming__c here

Comment (Julian88Tex) [reply robot/Cumulus/tests/browser/recurring_donations/check_recurring_auto_naming.robot]: @skristem thanks made that change!

**Pull request merged**